python-avd/pyavd/_eos_designs/schema/schema_fragments/bgp_peer_groups.schema.yml

```diff
@@ -34,6 +34,30 @@ keys:
             documentation_options:
               hide_keys: true
             $ref: "eos_cli_config_gen#/keys/router_bgp/keys/peer_groups/items"
+      use_separate_mlag_peer_group_for_overlay:
+        type: bool
+        default: false
+        description: Configure a different BGP peer group for MLAG for underlay and overlay.
```

Comment:
after discussion let's remove this boolean and rely on the name of `mlag_ipv4_vrfs_peer` being set different from the `mlag_ipv4_underlay_peer`

Reply:
Modified!